### PR TITLE
Nexys A7 - Ashling Opella-LD  connection details added

### PR DESCRIPTION
**Pull request by promodkumar-ashling**

### CODE REVIEW

File: emulation/quickstart/README.md [hunk 1]
```diff
@@ -52,8 +53,52 @@ Optional hardware includes:
...
+- Create a udev rules file to uniquely identify the Opella- LD device and set permissions as required by owner/ groups. An example udev file is provided:
+        Opella-LD\ Opella-LD_Linux_Rules_File\ [60-ashling.rules](https://www.ashling.com/wp-content/uploads/Opella-LD.zip) which identifies Opella-LD device (by Ashling’s USB product ID and Vendor ID).
+- The rules file must then be copied into the rules directory (requires root permission) e.g.: $ sudo cp ./60-ashling.rules /etc/udev/rules.d
+
+![Ashling Opella-LD-NexysA7 connection](NexysA7_Ashling_Opella-LD.png)
+
+
+- Connect Ashling Opella-LD to the bottom 6-pins of the JB connector. A 6- pin gender changer or flying leads can be used for this. Please make sure to match Vref of the Opella-LD and the JB connector. 
+
+Note: The connector used and the pinout can change based on the FPGA design.
```
Comment (MikeOpenHWGroup) [inline emulation/quickstart/README.md:80]: This table is useful information, but too detailed for a QSG.  I recommend moving it to the bottom of this file under the heading "Opella-LD pin-out" (or something like that).

**Review by MikeOpenHWGroup — CHANGES_REQUESTED** [review]
Hi @promodkumar-ashling, thanks for this update.  I've made a request to improve the flow of the document for novice users (the intended audience of this doc).  Please have a look.

File: emulation/quickstart/README.md (hunk 1 above)
Comment (promodkumar-ashling) [reply emulation/quickstart/README.md:80]: changes updated

Comment (MikeOpenHWGroup) [reply emulation/quickstart/README.md:80]: Thanks @promodkumar-ashling!

**Review by MikeOpenHWGroup — APPROVED** [review]

**Pull request merged**